PayPalMessaging/build.gradle

```diff
@@ -40,10 +41,19 @@ dependencies {
     implementation deps.coreKtx
     implementation deps.kotlinStdLib
     implementation deps.appCompat
+    implementation('com.paypal.messages:paypal-messages:1.0.0-alpha.00-SNAPSHOT')
 
     testImplementation deps.junitTest
 }
 
+allprojects {
```

Comment:
Can we add a TODO to remove this before feature release?

Reply:
✅ : https://github.com/braintree/braintree_android/pull/888/commits/a66d782dd41b374f932d83b898b76b49add187c5